kqueen_ui/asset/dynamic/scss/main.scss

```diff
@@ -552,6 +557,11 @@ table {
         color: $mira-gray-dark;
       }
     }
+    &.clusters-list, &.provisioners-list {
+      .action_column, .status_column {
+        width: 70px;
```

Comment:
please, remove this we should set width in a global level (in a separate commit)